src/main/java/com/sumologic/client/Credentials.java

```diff
@@ -7,37 +7,62 @@
  * @author Sebastian Mies
  */
 public class Credentials {
-    private String email;
-    private String password;
+    private String accessId;
+    private String accessKey;
 
     /**
-     * Constructs the credentials using email and password.
+     * Constructs the credentials using accessId and accessKey.
      *
-     * @param email Your email
-     * @param password Your password
+     * @param accessId Your access id
+     * @param accessKey Your access key
      */
-    public Credentials(String email, String password) {
-        this.email = email;
-        this.password = password;
+    public Credentials(String accessId, String accessKey) {
```

Comment:
I would not recommend doing this.  The signature is identical, which means compilation will succeed with no warnings at all and users will get unexpected failure at runtime.  It will be very difficult to diagnose.

Suggested approach: Create a  small class `AccessIdAndToken` and a new ctor `public Credentials(AccessIdAndToken)`.  Deprecate the old ctor.  That way you get a clear warning at compile-time, which you either fix right away, or at least is easier to find later when debugging.